themes/thecyberden.omp.json

```diff
@@ -0,0 +1,128 @@
+{
+    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+    "final_space": false,
+    "console_title": false,
+    "console_title_style": "folder",
+    "blocks": [
+        {
+            "type": "prompt",
+            "alignment": "left",
+            "horizontal_offset": 0,
```

Comment:
these are default values and can be removed

Reply:
Removed:
            "horizontal_offset": 0,
            "vertical_offset": 0,